pkg/framework/test/etcd.go

```diff
@@ -43,8 +43,29 @@ type SimpleSession interface {
 
 type simpleSessionStarter func(command *exec.Cmd, out, err io.Writer) (SimpleSession, error)
 
-// NewEtcd constructs an Etcd Fixture Process
-func NewEtcd(pathToEtcd string, config *EtcdConfig) *Etcd {
+var etcdBinPathFinder = DefaultBinPathFinder
+
+// NewEtcd returns a Etcd process configured with sane defaults
+func NewEtcd() (*Etcd, error) {
+	starter := func(command *exec.Cmd, out, err io.Writer) (SimpleSession, error) {
+		return gexec.Start(command, out, err)
+	}
+
+	config, err := NewEtcdConfig()
+	if err != nil {
+		return nil, err
+	}
+
+	return &Etcd{
```

Comment:
Why don't we delay defaulting until `Start` is called?

Reply:
Just because we wanted to refactor "all the things" (esp. #162) first. We can change that once this is done.